queries.sql

```diff
@@ -12,7 +12,46 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 
 SELECT * FROM animals WHERE neutered = true;
 
-SELECT * FROM ANIMALS WHERE name NOT LIKE '%Gabumon%';
+SELECT * FROM animals WHERE name NOT LIKE '%Gabumon%';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+UPDATE animals SET species = 'unspecified' WHERE species IS NULL;
+
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT animals;
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg > 0 OR weight_kg  > 0;
+
+ROLLBACK TO animals;
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
```

Comment:
- [ ] Please kindly start the transaction with `BEGIN;` 😉
- [ ] Also please commit the transaction at the end 😉